src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp

```diff
@@ -617,7 +617,7 @@ bool AieTracePlugin::configureStartIteration(xaiefal::XAieMod& core)
     std::string metricsStr = xrt_core::config::get_aie_trace_metrics();
```

Comment:
Copyright date needs to be updated on line 2.

Reply:
Done